odmlui/CommandManager.py

```diff
@@ -33,13 +33,14 @@ def undo(self):
         if len(self.undo_stack) == 0:
             self.enable_undo(enable=False)
         self.enable_redo()
-        e = None
+        failed = False
         try:
             cmd.undo()
         except Exception as e:
+            failed = True
             self.error_func(cmd, e)
 
-        if e:
+        if failed:
             raise
```

Comment:
Why not just raise inside the `except` block?

Reply:
I had the same doubt :sweat_smile: . I thought it might be a required behaviour that you guys would have decided initially (maybe for some extra handling? ). So, I didn't change the flow, just fixed the "Undefined Variable `e`" behaviour. 
I would fix this.